### PR TITLE
feat: styled the sidebar. updated sidebar headers.

### PR DESCRIPTION
**Pull request by keithgibson**
## Description

Styled the sidebar.
Changed top header to 'Styles", as it is not yet an Editor.
Changed the Regular rules header to be the source file name rather than the path.

## Checklist

- [ ] I've followed the [Contributing guidelines](https://github.com/community-health-demo/.github/blob/main/CONTRIBUTING.md)
- [x] I've titled my PR according to the [Conventional Commits spec](https://conventionalcommits.org)
- [ ] I've linked an open issue
- [ ] I've added tests that fail without this PR but pass with it
- [x] I've linted and tested my code
- [x] I've updated documentation (if appropriate)

### CODE REVIEW

File: client/components/RulesAllComp.jsx
Comment (mikebasta) [inline client/components/RulesAllComp.jsx:1]: During testing I noticed that if the target application will crash if it does not have source maps enabled. Since this is outside the scope of your PR, I created issue OSP-69 in Jira

https://mikebasta.atlassian.net/browse/OSP-69?atlOrigin=eyJpIjoiYmI5Y2ZiZDNkYWVkNGFmNGE2YjI4YTRiZjg5ZTQxOTIiLCJwIjoiaiJ9

**Review by mikebasta — APPROVED** [review]
Tested and working great. Very helpful comments, nice work!

File: client/components/RulesAllComp.jsx
Comment (keithgibson) [reply client/components/RulesAllComp.jsx:1]: Ah. Good catch. I just checked, and that's what caused the crash Saturday AM when we tried it with shopster. I enabled source maps in shopster and now it works with CSSxe.

**Review by Elena-Netepenko — APPROVED** [review]
